src/utils/error/handler.js

```diff
@@ -2,17 +2,22 @@ const errors = require("./errors");
 const httpStatus = require("../httpStatus");
 
 module.exports = (err, req, res, next) => {
-  
-  const { status, message } = errors.getErrorInfo(
-    err.name || err.message
-  );
+  const { status, message } = errors.getErrorInfo(err.name || err.message);
 
+  const statusMessage;
 
-  const statusMessage = httpStatus.getStatusText(status);
+  if (status === "MongoError") {
```

Comment:
Does this work? Wouldn't the status in this case always end up as 500 for this?

Reply:
status is actually the error name in this case.

Reply:
Why is it error name? If I recall that function returns a UNKNOWN_ERROR if it can't find the error you pass it. I didn't see you add a new entry for it. Am I missing something here?

Reply:
hmm, maybe i need some more work on this.

Reply:
ooh i read it wrong